src/main/python/run_regression.py

```diff
@@ -0,0 +1,142 @@
+"""
+Anserini: An information retrieval toolkit built on Lucene
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from __future__ import print_function
```

Comment:
The requirement is Python>=3.5 so this is not needed?

Reply:
Yea, right....
I think I am going to also test Python 2.7 to see if it works.

Reply:
refactor so that Python>=2.6 is supported